app/_src/production/use-kuma.md

```diff
@@ -0,0 +1,147 @@
+
+Kuma ships with a **read-only** HTTP API that you can use to retrieve Kuma resources.
+
+By default the HTTP API listens on port `5681`. To access Kuma we need to first port-forward the API service with:
+
+```sh
+kubectl port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
+```
+
+And then you can navigate to [`127.0.0.1:5681`](http://127.0.0.1:5681) to see the HTTP API.
+
+{% endtab %}
+{% tab use-kuma-kubernetes kumactl (Read-Only) %}
+
+You can use the `kumactl` CLI to perform **read-only** operations on Kuma resources. The `kumactl` binary is a client to the Kuma HTTP API, you will need to first port-forward the API service with:
+
+```sh
+kubectl port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
+```
+
+and then run `kumactl`, for example:
+
+```sh
+kumactl get meshes
+# NAME          mTLS      METRICS      LOGGING   TRACING
+# default       off       off          off       off
+```
+
+You can configure `kumactl` to point to any zone `kuma-cp` instance by running:
+
+```sh
+kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
+```
+
+{% endtab %}
+{% endtabs %}
+
+You will notice that Kuma automatically creates a [`Mesh`](/docs/{{ page.version }}/policies/mesh) entity with name `default`.
+
+## Use {{site.mesh_product_name}} with Universal
+
+{% tabs use-kuma-universal useUrlFragment=false %}
+{% tab use-kuma-universal GUI (Read-Only) %}
+
+Kuma ships with a **read-only** GUI that you can use to retrieve Kuma resources. By default the GUI listens on the API port and defaults to `:5681/gui`.
+
+To access Kuma you can navigate to [`127.0.0.1:5681/gui`](http://127.0.0.1:5681/gui) to see the GUI.
+
+{% endtab %}
+{% tab use-kuma-universal HTTP API (Read & Write) %}
+
+Kuma ships with a **read and write** HTTP API that you can use to perform operations on Kuma resources. By default the HTTP API listens on port `5681`.
+
+To access Kuma you can navigate to [`127.0.0.1:5681`](http://127.0.0.1:5681) to see the HTTP API.
+
+{% endtab %}
+{% tab use-kuma-universal kumactl (Read & Write) %}
+
+You can use the `kumactl` CLI to perform **read and write** operations on Kuma resources. The `kumactl` binary is a client to the Kuma HTTP API. For example:
+
+```sh
+kumactl get meshes
+# NAME          mTLS      METRICS      LOGGING   TRACING
+# default       off       off          off       off
+```
+
+or you can enable mTLS on the `default` Mesh with:
+
+```sh
+echo "type: Mesh
+name: default
+mtls:
+  enabledBackend: ca-1
+  backends:
+  - name: ca-1
+    type: builtin" | kumactl apply -f -
+```
+
+You can configure `kumactl` to point to any zone `kuma-cp` instance by running:
+
+```sh
+kumactl config control-planes add --name=XYZ --address=http://{address-to-kuma}:5681
+```
+{% endtab %}
+{% endtabs %}
+
+You will notice that Kuma automatically creates a [`Mesh`](/docs/{{ page.version }}/policies/mesh) entity with name `default`.
```

Comment:
Do we need to duplicate this sentence under both sections? Can we just have one?

Reply:
@slonka I think they're both needed since someone might only read one section if they only need the Kubernetes or the Universal instructions, right?

Reply:
I meant like one under both sections we could have something like 

{% tip %}
You will notice that Kuma automatically creates a [`Mesh`](/docs/{{ page.version }}/policies/mesh) entity with name `default`.
{% endtip %}

that is relevant for both sections. But this is just a nitpick, feel free to ignore this if you don't think this would improve the doc.